File: MealPlanService/MealPlanService.Application/DTOs/NutrientOfDayDTO.cs
using MealPlanService.Domain.Enums;

namespace MealPlanService.Application.DTOs
{
    public class NutrientOfDayDTO
    {
        public NutrientType NutrientType { get; set; }
        public CalculationType CalculationType { get; set; }
        public double? Value { get; set; }
    }
}

File: NutritionBalance.Client/src/app/models/mealPlan/DTOs/nutrient-of-day.dto.ts
import { NutrientType } from '../enums/nutrient-type.enum';
import { CalculationType } from '../enums/calculation-type.enum';

export class NutrientOfDayDTO {
  nutrientType!: NutrientType;
  calculationType!: CalculationType;
  value?: number;
}
